tortoise/tortoise_test.go

```diff
@@ -2146,23 +2140,22 @@ func TestOnBallotComputeOpinion(t *testing.T) {
 
 		id := types.BallotID{1}
 		ballot := types.NewExistingBallot(id, nil, nil, rst[0].InnerBallot)
-		ballot.Votes.Base = types.GenesisBallotID
-		ballot.Votes.Support = []types.BlockID{types.GenesisBlockID}
+		ballot.Votes.Base = types.EmptyBallotID
+		ballot.Votes.Support = nil
 		ballot.Votes.Against = nil
 
 		tortoise.OnBallot(&ballot)
 
 		info := tortoise.trtl.ballotRefs[id]
 		hasher := hash.New()
-		hasher.Write(types.GenesisBlockID[:])
 		for i := 0; i < distance-1; i++ {
 			buf := hasher.Sum(nil)
 			hasher.Reset()
 			hasher.Write(buf)
 		}
 		require.Equal(t, hasher.Sum(nil), info.opinion().Bytes())
 	})
-	t.Run("support abstain support", func(t *testing.T) {
+	t.Run("against abstain support", func(t *testing.T) {
```

Comment:
NIT, It took me a while to understand this title. How about:
```suggestion
	t.Run("against → abstain → support", func(t *testing.T) {
```

Reply:
i am not sure that i like additional ->